tex/artikel/jDPG.tex

```diff
@@ -5,81 +5,83 @@
 \begin{minipage}{0.75\textwidth}
     \centering
     \textbf{Liebe Physik-Erstis der Uni Münster,\\
-    auch die junge DPG möchte Euch herzlich im Kreis der Physik-Studenten begrüßen.}
+    auch die junge DPG möchte Euch herzlich im Kreis der Physikstudierenden begrüßen.}
 \end{minipage}\hfill
 \begin{minipage}{0.1\textwidth}
     \includegraphics[width=\textwidth]{res/jdpg_logo.pdf}
 \end{minipage}
 
 \begin{multicols}{2}
-In Deutschland studieren rund \num{30000} Studenten Physik.
+In Deutschland studieren rund \num{30000} Studierende Physik.
 Du bist jetzt einer von ihnen.
-An "Jungphysiker" wie dich wendet sich die junge Deutsche Physikalische Gesellschaft~(jDPG) mit ihren Angeboten rund um das Physikstudium.
+An "Jungphysiker:innen" wie dich wendet sich die junge Deutsche Physikalische Gesellschaft~(jDPG) mit ihren Angeboten rund um das Physikstudium.
```

Comment:
Als Anmerkung, ich habe bisher mit "*" gegendert, dieser Artikel benutzt ":". Sollten wir das noch vereinheitlichen?

Reply:
Ich würde sagen, weil der Artikel von der jDPG kommt, ist so ein kleiner Stilbruch vollkommen in Ordnung?